scanner/services/indexer.go

```diff
@@ -18,10 +18,13 @@ import (
 	"github.com/stackrox/rox/pkg/grpc/authz/allow"
 	"github.com/stackrox/rox/scanner/indexer"
 	"google.golang.org/grpc"
+	grpchealth "google.golang.org/grpc/health/grpc_health_v1"
 )
 
 type indexerService struct {
+	grpchealth.UnimplementedHealthServer
```

Comment:
From the ADR I took you were going to create one Health service endpoint, which should perform different checks based on the mode Scanner is running on.

> Both modes of Scanner v4 execution, Indexer and Matcher, will expose this service (ScannerDB has no need for this). However, both modes will define "ready" (ie `status` of `SERVING`) differently.

If I understood correctly, what you are doing is augmenting each gRPC service (Indexer and Matcher) to offer the health RPCs. How the Kubelet would probe this when running Scanner in the Central Cluster? Two gRPC calls?